In [1]:
import matplotlib.pyplot as plt
import numpy as np
import pandas as pd
import seaborn as sns
import utils
%matplotlib inline
df = utils.wrangle()

In [2]:
df[df.columns[4:-1]].corr()

FFMC       DMC        DC       ISI      temp        RH  \
FFMC       1.000000  0.382619  0.330512  0.531805  0.431532 -0.300995   
DMC        0.382619  1.000000  0.682192  0.305128  0.469594  0.073795   
DC         0.330512  0.682192  1.000000  0.229154  0.496208 -0.039192   
ISI        0.531805  0.305128  0.229154  1.000000  0.394287 -0.132517   
temp       0.431532  0.469594  0.496208  0.394287  1.000000 -0.527390   
RH        -0.300995  0.073795 -0.039192 -0.132517 -0.527390  1.000000   
wind      -0.028485 -0.105342 -0.203466  0.106826 -0.227116  0.069410   
rain       0.056702  0.074790  0.035861  0.067668  0.069491  0.099751   
area       0.040122  0.072994  0.049383  0.008258  0.097844 -0.075519   
log(area)  0.046799  0.067153  0.066360 -0.010347  0.053487 -0.053662   

               wind      rain      area  log(area)  
FFMC      -0.028485  0.056702  0.040122   0.046799  
DMC       -0.105342  0.074790  0.072994   0.067153  
DC        -0.203466  0.035861  0.049383   0.066360  
ISI        0.106826  0.067668  0.008258  -0.010347  
temp      -0.227116  0.069491  0.097844   0.053487  
RH         0.069410  0.099751 -0.075519  -0.053662  
wind       1.000000  0.061119  0.012317   0.066973  
rain       0.061119  1.000000 -0.007366   0.023311  
area       0.012317 -0.007366  1.000000   0.524134  
log(area)  0.066973  0.023311  0.524134   1.000000

Correlation coefficients are quite low all around. The highest any two predictors get in correlation is 0.68, which is still quite low in terms of colinearity. The highest correlation between the predictors and either area or log(area) is 0.098, which extremely low. 